src/SFA.DAS.LevyTransferMatching.Web/Views/Opportunities/Detail.cshtml

```diff
@@ -5,57 +5,49 @@
+                            <div class="govuk-radios__item">
+                                <input class="govuk-radios__input" id="apply-pledge-funds-yes" asp-for="HasConfirmed" value="true" type="radio" data-aria-controls="conditional-apply-pledge-funds-yes" das-highlight-error-for="HasConfirmed" error-class="govuk-textarea--error">
+                                <label class="govuk-label govuk-radios__label" for="apply-pledge-funds-yes">Yes, apply for transfer funds</label>
+                            </div>
+                            <div class="govuk-radios__conditional govuk-radios__conditional--hidden" id="conditional-apply-pledge-funds-yes">
+                                <div class="govuk-form-group">
+                                    <div class="govuk-warning-text">
+                                        <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+                                        <strong class="govuk-warning-text__text">
+                                            <span class="govuk-warning-text__assistive">Warning</span>
+                                            You'll need an apprenticeship service account to apply to be considered for funding
+                                        </strong>
                                     </div>
                                 </div>
-                                <div class="govuk-radios__item">
-                                    <input class="govuk-radios__input" id="apply-pledge-funds-no" asp-for="HasConfirmed" value="false" type="radio" das-highlight-error-for="HasConfirmed" error-class="govuk-textarea--error">
-                                    <label class="govuk-label govuk-radios__label" for="apply-pledge-funds-no">No</label>
-                                </div>
                             </div>
-                        </fieldset>
-                    </div>
-
+                            <div class="govuk-radios__item">
+                                <input class="govuk-radios__input" id="apply-pledge-funds-no" asp-for="HasConfirmed" value="false" type="radio" das-highlight-error-for="HasConfirmed" error-class="govuk-textarea--error">
+                                <label class="govuk-label govuk-radios__label" for="apply-pledge-funds-no">No</label>
+                            </div>
+                        </div>
+                    </fieldset>
+                </div>
+                <div class="govuk-button-group">
                     <button class="govuk-button" data-module="govuk-button" id="apply-application-continue">Continue</button>
-
-                    <a href="@Url.Action("Index")" class="govuk-link govuk-link-secondary govuk-link--no-visited-state govuk-back-link">Back</a>
-
-                </form>
-
-            </div>
+                    <a href="@Url.Action("Index")" class="govuk-back-link">Back</a>
```

Comment:
one of my PRs raised not having back as the text for a link and to use something more descriptive, not sure if this is applicable in this instance, you'll have a better idea than me

Reply:
This is fine in this instance